.github/workflows/chart.yaml

```diff
@@ -4,8 +4,12 @@ name: Chart
 on:
   push:
     branches: [main]
+    paths:
+      - 'charts/**'
```

Comment:
the chart tests also build and install the container images so these tests could in theory also fail if we change things in other paths

as such they currently seem to serve as simple full stack smoketests, maybe we want to consider keeping them on all changes for that purpose?

Reply:
i remove the condition from the charts workflow, but why i see that the tests are still `Waiting for status to be reported`

Reply:
@hairmare do you know why there is three frontend tests are there even if we define the job to be executed only if there is a change in the `frontend` folder ?

Reply:
our rules still need them to succeed

Reply:
![image](https://github.com/adfinis/timed/assets/110528300/38cef6e3-bc94-4f88-adc0-fee650148de3)

Reply:
there is no way to define the required jobs to be defined in the file ?

Reply:
there are probably ugly workarounds

Reply:
i will keep this PR open, since there are other tasks more important than this, we can come to it later

Reply:
```yaml
# without paths
...
jobs:
  dirty:
     outputs:
       dirty: ${{ steps.check-dirty.outputs.dirty }}
     steps:
      - id: dirty
        run: |
          # bash script to determine if dirty, then write to output

  lint:
      needs: dirty
      steps:
       - name: foo
         if: ${{ needs.dirty.outputs.dirty }}
         run: |
           ls -a
```

something like this would probably work, but is very ugly

Reply:
i totally agree that it's ugly

Reply:
I'm not sure if it works, but could you try the inverse of `paths`.
* For the **backend** workflow:
```yml
  pull_request:
    paths-ignore:
      - 'frontend/**'
```
* For the **frontend** workflow:
```yml
  pull_request:
    paths-ignore:
      - 'backend/**'
```